amnestywebsite/humanity-theme
# Refactor sidebar into a block

Ref: https://github.com/amnestywebsite/humanity-theme/issues/246

**Steps to test**:
1. make sure you have a default global sidebar enabled in theme options
2. create a post
3. preview it
4. the default global sidebar should be rendered
5. in appearance options, pick a different sidebar and save
6. reload the frontend
7. the chosen sidebar should render instead
